### ISSUE TITLE
Unable to `sudo apt install -y just` from Github Actions

### ISSUE THREAD

**Issue by be-rock** [casey/just#2432]
Hi,

Thanks for the great project.

I am unable to `sudo apt install -y just` from Github Actions. I have tried both an `ubuntu-20.04` and `ubuntu-22.04` image and both result in:

```
E: Unable to locate package just
Error: Process completed with exit code 100.
```

Possibly related: https://github.com/casey/just/issues/1521#issue-1551853841

Thanks.

**Comment by be-rock** [comment]
I wonder if there is an issue with the docs

https://docs.makedeb.org/prebuilt-mpr/getting-started/#setting-up-the-repository

This `wget` command doesn't seem to return anything:

```
wget -qO - 'https://proget.makedeb.org/debian-feeds/prebuilt-mpr.pub' ; echo $?
4
```

**Comment by casey** [comment]
Unfortunately `just` is only available for Ubuntu 24.04 and later. Soon!
